uniflow-android/src/main/java/io/uniflow/android/livedata/LiveDataPublisher.kt

```diff
@@ -27,7 +27,9 @@ open class LiveDataPublisher(
     override suspend fun publishState(state: UIState, pushStateUpdate: Boolean) {
         onMain(immediate = true) {
             UniFlowLogger.debug("$tag --> $state")
-            _states.value = state
+            if (pushStateUpdate) {
```

Comment:
This will need to be changed because it will break logic for 

// initial state - UIState.Empty
publishState(CardState(), false)
// from the PR, I am not sure that's the following passes. It will be nice to have some unit tests to uniflow for this
getState() == CardState()